src/docs/arch-tiered-storage.dox

```diff
@@ -0,0 +1,440 @@
+/*! @arch_page arch-tiered-storage Tiered Storage
+
+\warning Tiered storage is considered an experimental part of the WiredTiger
+library, and is not yet ready for production workloads.
+
+@section ts_intro Introduction and Definitions
+
+Tiered storage allows B-Trees to be stored into multiple places, more
+recently updated blocks on local disk and less recently updated blocks in cloud storage.
+The term \a object refers to one of these written
+btree parts, whether the object resides on the local disk, as an
+<em>object file</em>, or in the cloud, as a <em>cloud object</em>.
+The mechanism to create new objects is a WT_SESSION::checkpoint API call
+with the \c flush_tier configuration set.
+For brevity, is often called a \c flush_tier call.
```

Comment:
Missing word: "For brevity, it is called.." or something.

Reply:
Fixed